README.md

```diff
@@ -709,9 +711,9 @@ This method of upload can be used to upload a large number of files and folders
 -   Folder names that conflict with a FILE name at the destination will cause an error to be thrown
 -   Folder names that conflict with a FOLDER name at the destination will use the existing folder ID (i.e. skip) rather than creating a new folder
 -   File names that conflict with a FOLDER name at the destination will cause an error to be thrown
--   File names that conflict with a FILE name at the destination will be SKIPPED
+-   When the intended file name conflicts with a FILE name at the destination the file will be SKIPPED if the last modified date matches the file to upload. If they have different last modified dates, it will be uploaded as a new REVISION
 
-Similar to the single file upload, the above FILE to FILE name conflict resolution behavior can be modified by the `--replace` command. This will force new revisions on all conflicts within the bulk upload.
+Similar to the single file upload, the above FILE to FILE name conflict resolution behavior can be modified. Use the `--replace` option to will force new revisions on all conflicts within the bulk upload regardless of last modified date. Or use the `--skip` option to simply skip all FILE to FILE name conflicts.
```

Comment:
Grammar problem here

Reply:
> to simply skip all FILE to FILE name conflicts.
Will that not apply to FILE/FOLDER name conflicts as well?

Reply:
Good catch! 

I've added that functionality in the latest commits. The single file update was pretty straightforward.

When adding this for the bulk side, I took the opportunity to refactor the `checkAndAssignNames` function out of the ArDrive class and into the folder wrapper class. There was nothing keeping it there and I think the method makes more sense within the folder scope.

In doing this, I lifted all of the error throwing into the recursive method so I could properly `skip` them if the conflict resolution setting was appropriate.